CASMTRIAGE-7106 adding procedure for building NCN images in CSM 1.5.2 patch (#5190)
Cray-HPE/docs-csm: upgrade/1.5.2/README.md

1. Choose one of the following options based on the output from the previous step.

    * Option 1: [Upgrade of CSM on system with additional products](./CSM-With-Other-Products.md)
    * Option 2: [Upgrade of CSM on CSM-only system](./CSM-Only.md#steps) _(Do not use this procedure if more than CSM is

>       installed on the system.\)_

Author: What is the correct formatting for these underscores? Checks say they are misspelled

Reviewer: Maybe try replacing them with asterisks or double asterisks. Oh, or maybe it just doesn't like them being spread across multiple lines like that...

Reviewer: Yeah, moving the _-surrounded text to the same line fixed the lint complaint

>     * Option 2: [Upgrade of CSM on CSM-only system](./CSM-Only.md#steps) _(Do not use this procedure if more than CSM is
>       installed on the system.\)_

Reviewer: ```suggestion
    * Option 2: [Upgrade of CSM on CSM-only system](./CSM-Only.md#steps)
      _(Do not use this procedure if more than CSM is installed on the system.\)_
```

These clusters are automatically backed up every 24 hours, but taking a manual backup at this stage in the upgrade
enables restoring from backup later in this process if needed.

### NCN upgrade

> This step is necessary so that nodes are using the correct images after running [Update NCN images](../1.5.2/README.md#update-ncn-images).

Reviewer 2: The link to the upgrade page has not been added here. Is this fine?

Author: Great catch. I have added the link.

These clusters are automatically backed up every 24 hours, but taking a manual backup at this stage in the upgrade
enables restoring from backup later in this process if needed.

### NCN upgrade

> This step is necessary so that nodes are using the correct images. If NCN are
> not upgraded into these images, then `cloudinit` would fail on the nodes the next
> time the nodes are rebuilt. The images that the nodes are being upgraded to were
> set in BSS during [Update NCN images](../1.5.2/README.md#update-ncn-images).
> If the NCN nodes are not upgraded now, then any node reboot will cause the node to be
> booted into this new image which is not following the proper upgrade procedure which could cause problems.

Reviewer 3: I think we can shorten the language here to be more to the point.
```suggestion
This step is necessary so that nodes are using the correct images after running [Update NCN images](../1.5.2/README.md#update-ncn-images).
```

> Additionally, each patch release includes updated container images that may
> contain CVE fixes, rebuilding each NCN will refresh cached container images.
> For instructions on how to upgrade NCNs for this patch release, refer to [Upgrade NCN during CSM `1.5.2` patch](./Upgrade_NCN_images.md).

Reviewer 3: ```suggestion
The rebuild will also ensure that the NCN has the latest cached container images that often accompany a CSM patch release.
```
